analyzer-models/src/test/java/ffb/test/utilities/GenericTests.java

```diff
@@ -0,0 +1,52 @@
+package ffb.test.utilities;
+
+import ffb.analyzer.models.espn.TransactionCounter;
+import org.junit.Assert;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class GenericTests {
```

Comment:
How about `GenericTestUtils`?

Reply:
I initially didn't want to add util to the name because it's already in a utilities package, but looking at it again the name makes it seem like a test. I'll update it to GenericTestUtils for now, but I think we can come up with a better name eventually.

Reply:
I think we should move it into our `.core` package, with the `DateUtils` class in my PR. We can cut down on the dependencies we'll have to include in each package. But it's fine for now.